<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBTWTarief</class>
 <widget class="QDialog" name="AddBTWTarief">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beheer BTW tarieven</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>58</y>
     <width>591</width>
     <height>76</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>76</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="tbPercentage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbPercentage</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPercentage">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Percentage::</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTarief">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tarief (omschr.):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbTarief">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>10</y>
      <width>166</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbTarief</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbVanaf">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BTW geldig vanaf:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpVanaf">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpTot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbTot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>45</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tot:</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tvBTWTarief">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>139</y>
     <width>591</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>591</width>
     <height>47</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnNieuw">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBewerk">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/edit_button.png</normaloff>:/icons/icons/edit_button.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpslaan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>226</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
